### PR TITLE
[docs] README conda install

### PR DESCRIPTION
**Pull request by aclegg3**
## Motivation and Context

Introducing conda install on this release, so tell new users how to use this for install.

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Skylion007** [comment]
@aclegg3 How is this different from #588 ?

**Comment by aclegg3** [comment]
> @aclegg3 How is this different from #588 ?

Thanks, didn't realize this was open and waiting. Thanks for linking. We didn't have nightly build yet, so we were waiting. 🤷

File: README.md [hunk 1]
```diff
@@ -152,6 +152,15 @@ To run the above benchmarks on your machine, see instructions in the [examples](
 
 ## Installation
 
+### Conda Packages
+
+We provide conda packages for [the most recent release](https://anaconda.org/aihabitat) and built [nightly for the master branch](https://anaconda.org/aihabitat-nightly). Packages are provided for standard and headless systems with and without Bullet physics support. When possible, we suggest this method of installation for the most simple and straight-forward installation experience. To learn more about building your own conda packages for Habitat-sim, checkout the [conda-build README](https://github.com/facebookresearch/habitat-sim/tree/master/conda-build).
```
Comment (erikwijmans) [inline README.md:157]:
```suggestion
We provide conda packages for [the most recent release](https://anaconda.org/aihabitat) and built [nightly for the master branch](https://anaconda.org/aihabitat-nightly). Packages are provided for standard and headless systems with and without Bullet physics support. When possible, we suggest this method of installation for the most simple and straight-forward installation experience.
```

That readme is just for building habitat sim's conda package. 99% of users don't need to read it.

File: README.md [hunk 2]
```diff
@@ -152,6 +152,15 @@ To run the above benchmarks on your machine, see instructions in the [examples](
...
+
+Example of installing conda package for master branch with display system and Bullet physics support:
+```
+conda install -c aihabitat-nightly withbullet
```
Comment (erikwijmans) [inline README.md:161]: To install habitat-sim on systems with an attached display:
```
conda install habitat-sim  -c conda-forge -c aihabitat
```

To install the headless build of habitat sim
```
conda install habitat-sim headless -c conda-forge -c aihabitat
```

To install habitat-sim with bullet physics [on a headless system]
```
conda install habitat-sim withbullet [headless] -c conda-forge -c aihabitat
```

To get the nightly build of latest master, simply swap `-c aihabitat` for `-c aihabitat-nightly`

**Review by erikwijmans — CHANGES_REQUESTED** [review]

**Review by dhruvbatra — APPROVED** [review]

**Review by erikwijmans — APPROVED** [review]

**Review by Skylion007 — APPROVED** [review]

**Pull request merged**